tanner/server.py

```diff
@@ -126,20 +128,28 @@ def create_app(self, loop):
         return app
 
     async def start_background_delete(self, app):
-        app['session_delete'] = asyncio.ensure_future(self.delete_sessions())
+        app["session_delete"] = asyncio.ensure_future(self.delete_sessions())
 
     async def cleanup_background_tasks(self, app):
-        app['session_delete'].cancel()
-        await app['session_delete']
+        app["session_delete"].cancel()
+        await app["session_delete"]
 
     def start(self):
         loop = asyncio.get_event_loop()
-        self.redis_client = loop.run_until_complete(redis_client.RedisClient.get_redis_client())
+        self.redis_client = loop.run_until_complete(
+            redis_client.RedisClient.get_redis_client()
+        )
+        self.pg_client = loop.run_until_complete(
+            postgres_client.PostgresClient().get_pg_client()
```

Comment:
I run the tanner without postgresql running and it doesn't exit with error. Could you please check?

Reply:
You have the Postgres in your config, right? 
I mean I just tested without running Postgres and I got the error 

```
psycopg2.OperationalError: could not connect to server: Connection refused
        Is the server running on host "localhost" (::1) and accepting
        TCP/IP connections on port 5432?
could not connect to server: Connection refused
        Is the server running on host "localhost" (127.0.0.1) and accepting
        TCP/IP connections on port 5432?
```
I did not make any changes to the code or anything I just simply stopped my Postgres and ran `sudo python bin/tanner`